core/assets/bundles/bundle_fr.properties

```
@@ -99,6 +99,7 @@ mod.enabled = [lightgray]Activé
 mod.disabled = [scarlet]Désactivé
 mod.disable = Désactiver
 mod.delete.error = Unable to delete mod. File may be in use.
+mod.requiresversion = [scarlet]Version du jeu requise: [accent]{0}
```

A space must be before a `:`.
```suggestion
mod.requiresversion = [scarlet]Version du jeu requise : [accent]{0}
```